security/pfSense-pkg-suricata/files/usr/local/pkg/suricata/suricata.inc

```diff
@@ -4255,26 +4169,26 @@ function suricata_get_vpns_list() {
 		require_once("ipsec.inc");
 	}
 	if (ipsec_enabled()) {
-		if (is_array($config['ipsec']['client']) && isset($config['ipsec']['client']['enable'])) {
+		if (config_get_path('ipsec/client') && config_path_enabled('ipsec/client')) {
```

Comment:
```suggestion
		if (config_path_enabled('ipsec/client')) {
```
`config_path_enabled('ipsec/client') => true` implies `config_get_path('ipsec/client') => true`

Reply:
Ah...I missed that one in a previous look. Made the suggested change.